docs/src/dist-training-torch/ddp_intro.md

```diff
@@ -0,0 +1,63 @@
+
+# Introduction to DDP Training with Lance
+
+
+Lance is a modern columnar format designed for high-performance ML workloads. Its architecture enables fast random access and efficient scans. LanceDB's ecosystem provides tools that integrate this storage format with training frameworks, supporting the AI workflow from data storage to optimized training pipelines. 
+
+This guide focuses on the PyTorch integration and demonstrates how to use Lance to build efficient data loaders for Distributed Data Parallel (DDP) training.
+
+## Data Loading Strategies: Map-Style vs. Iterable-Style
+
+PyTorch offers two paradigms for creating datasets: map-style and iterable-style. Lance provides a corresponding class for each.
+
+### Map-Style: `torch.utils.data.Dataset`
+A map-style dataset is one that can be indexed and has a known length. It must implement the `__getitem__(self, index)` and `__len__(self)` methods. This design allows the `DataLoader` to fetch any specific item from the dataset by its index, enabling features like random shuffling and straightforward parallel loading across multiple workers. 
+
+*   **Lance Integration:** `lance.torch.data.SafeLanceDataset`
+*   **Best For:** Finite, indexable datasets. This covers the vast majority of standard use cases, such as image classification (ImageNet, FOOD101), text classification, or any scenario where the dataset is a discrete collection of samples.
+
+### Iterable-Style: `torch.utils.data.IterableDataset`
+An iterable-style dataset works like a Python generator. It implements the `__iter__(self)` method, which yields data items sequentially as a stream. It often has no known length and does not support random access to a specific index. This makes it ideal for handling data that doesn't fit the indexed collection model.
+
+*   **Lance Integration:** `lance.torch.data.LanceDataset`
+*   **Best For:** True streaming data sources (e.g., from a database cursor or real-time log), extremely large datasets where building a full index is impractical, or when implementing complex, custom sampling logic that goes beyond what standard PyTorch samplers offer. 
```

Comment:
The `Best For` section here is a little confusing.  What is an infinite Lance dataset?  All Lance datasets can be either map style or iterable style.  You explain how to choose them in the next section, do you also need a "best for" here?

Reply:
yeah makes sense.. removed it from here